xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-rest/src/main/java/org/xwiki/search/solr/internal/rest/SOLRSearchSource.java

```diff
@@ -136,7 +136,7 @@
         // Boost
         // FIXME: take it from configuration
         query.bindValue("qf",
-            "title^10.0 name^10.0 doccontent^2.0 objcontent^0.4 filename^0.4 attcontent^0.4 doccontentraw^0.4 "
+            "title_^10.0 name^10.0 doccontent^2.0 objcontent^0.4 filename^0.4 attcontent^0.4 doccontentraw^0.4 "
```

Comment:
Are you sure about this change?

Reply:
Doesn't look good to me either.

Reply:
I wasn't totally sure about this, but there isn't actually a field in the schema called "title". Instead there's a field called "title_" which looks like it gets populated with all of the localized title fields. Here's the definition of that field in the solr schema:

https://github.com/xwiki/xwiki-platform/blob/d3ca366ca7ff9926d0ff25ecf89acaf697368d6a/xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-server/xwiki-platform-search-solr-server-data/src/main/resources/xwiki/conf/managed-schema#L170

Later in the schema there's a copyField from title_* to title_. So to boost a match against title, it seems like this is the way to go.

Reply:
See https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-server/xwiki-platform-search-solr-server-data/src/main/resources/xwiki/conf/solrconfig.xml#L766 . "title" is a shortcut for all "title_*" fields.

Reply:
Ok, that makes sense. I will revert that line to the original then. Thanks for the explanation.